.buildkite/snyk_report_pipeline.yml

```diff
@@ -1,7 +1,10 @@
 agents:
-  provider: "gcp"
-  machineType: "n1-standard-4"
-  image: family/core-ubuntu-2204
+  image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-logstash-ci-no-root"
+  cpu: "4"
+  memory: "16Gi"
+  ephemeralStorage: "128Gi"
+  # Run as non root (logstash) user. UID is hardcoded in image.
+  imageUID: "1002"
 
```

Comment:
Sorry @dliappis to make a work for you. For the Snyk report pipeline, I manually installed Java using the GCP BK agent and cloned the repo in the script. As we can use `buildkite-agent-logstash-ci-no-root` agent now, I have changed the agent and simplified the logic not to clone.
I would appreciate your review. Thanks.